src/components/VErrorSection/VNoResults.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="no-results text-center md:text-left">
-    <h5 class="break-words text-5xl">
+    <h1 class="break-words text-6xl md:text-4xl">
```

Comment:
```suggestion
    <h1 class="break-words text-4xl md:text-6xl">
```
Sorry for mixing the sizes up :)